### PR TITLE
Escape FQDN in documentation so it shows correctly

### PR DESCRIPTION
**Pull request by vitrus**
## This PR makes sure the slashes in a FQDN are escaped and thus displayed

Browsing through the documentation I got really confused by the mention of `AppAdminFooAdmin` where I would have expected `App\Admin\FooAdmin`. The missing slashes are in the `.rst` file but not escaped so not displayed on the Symfony documentation pages.

This PR fixes the escaping.

I am targeting this branch, because it is a documentation fix that should be backwards compatible.

Does not close any issue, at it is a simple documentation fix.

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you for your contribution, you are right, but the fix should be a bit different, I applied a suggestion to the line!

Also can you please rebase your branch with the latest changes from `3.x`?

**Comment by vitrus** [comment]
Hi @OskarStark, I initially did this PR on `master` but concluded from the PR template that it should have been done op 3.x, so changed that. 

I honestly don't know how to rebase it from 3.x branch (is there a button to do this?). Or shall I just create a new branch + PR ...

File: docs/reference/security.rst [hunk 1]
```diff
@@ -132,7 +132,7 @@ service), Sonata will check if the user has the ``ROLE_APP_ADMIN_FOO_EDIT`` or `
 
 .. note::
 
-    Declaring the same admin as `App\Admin\FooAdmin` results in
+    Declaring the same admin as `App\\Admin\\FooAdmin` results in
```
Comment (OskarStark) [inline docs/reference/security.rst]:
```suggestion
    Declaring the same admin as ``App\Admin\FooAdmin`` results in
```

**Comment by vitrus** [comment]
@OskarStark  You are right, will do it that way. 
Found 3 more issues (some other escaping things, and another issue)

I think I'll just open up a new PR...

**Pull request closed without merging**